src/EditorFeatures/Core/Implementation/InlineRename/InlineRenameSession.cs

```diff
@@ -171,7 +172,9 @@ private void CreateOpenTextBufferManagerForBuffer(ITextBuffer buffer, IEnumerabl
             AssertIsForeground();
             VerifyNotDismissed();
 
-            if (!_openTextBuffers.ContainsKey(buffer))
+            var documentSupportsRefactoringService = _workspace.Services.GetService<IDocumentSupportsSuggestionService>();
+
+            if (!_openTextBuffers.ContainsKey(buffer) && documents.All(d => documentSupportsRefactoringService.SupportsRename(d)))
```

Comment:
This now means as a side effect that `_openTextBuffers[buffer]` might not be initialized which some caller of this would expect. Is that bad? Specifically, what happens about the case for lines 163/164 above if the trigger span is in ASP.NET buffers?